test/fit_tests/deploy/rackhd_stack_init.py

```diff
@@ -121,7 +143,7 @@ def test05_discover_control_switch_node(self):
                          + str(api_data['status']))
```

Comment:
Just a note for later.  We need to add a check to make sure the switch was actually discovered correctly.